line-item-presets/amazonVideo.sample.json

```diff
@@ -0,0 +1,27 @@
+{
+  "orderId": "3170606300",
+  "creativeId": "138426662160",
+  "iterator": "0.25,0.50,0.75,1.00,1.25,1.50,1.75,10.00,10.25,10.50,10.75,11.00,11.25,11.50,11.75,12.00,12.25,12.50,12.75,13.00,13.25,13.50,13.75,14.00,14.25,14.50,14.75,15.00,15.25,15.50,15.75,16.00,16.25,16.50,16.75,17.00,17.25,17.50,17.75,18.00,18.25,18.50,18.75,19.00,19.25,19.50,19.75,2.00,2.25,2.50,2.75,20.00,20.50,21.00,21.50,22.00,22.50,23.00,23.50,24.00,24.50,25.00,25.50,26.00,26.50,27.00,27.50,28.00,28.50,29.00,29.50,3.00,3.25,3.50,3.75,30.00,31.00,32.00,33.00,34.00,35.00,36.00,37.00,38.00,39.00,4.00,4.25,4.50,4.75,40.00,41.00,42.00,43.00,44.00,45.00,46.00,47.00,48.00,49.00,5.00,5.25,5.50,5.75,50.00,6.00,6.25,6.50,6.75,7.00,7.25,7.50,7.75,8.00,8.25,8.50,8.75,9.00,9.25,9.50",
```

Comment:
The prices are not sorted - are you 100% they match the A9 price map? Can you sort it?

Reply:
fixed